primitive-types/tests/fp_conversion.rs

```diff
@@ -0,0 +1,78 @@
+// Copyright 2020 Parity Technologies
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+//! Tests for to and from f64 lossy for U256 primitive-type.
+
+#![cfg(feature = "fp-conversion")]
```

Comment:
see the bottom of the `primitive-types/Cargo.toml` for how it should be done
```toml
[[test]]
name = "fp_conversion"
required-features = ["fp-conversion"]
```

Reply:
Thanks, made the change here!